src/components/SecretsModal/Annotations.js

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { TextInput } from 'carbon-components-react';
+import Add from '@carbon/icons-react/lib/add--alt/24';
+import Remove from '@carbon/icons-react/lib/misuse/24';
+import './SecretsModal.scss';
+
+const Annotations = props => {
+  const {
+    annotations,
+    handleChange,
+    handleAdd,
+    handleRemove,
+    invalidFields,
+    disabled
+  } = props;
+
+  const annotationFields = [];
+  for (let i = 0; i < annotations.length; i += 2) {
+    annotationFields.push(
+      <div className="annotationRow" key={i}>
+        <TextInput
+          id={`annotation${i}`}
+          data-testid={`annotation${i}`}
+          labelText=""
+          key={`annotation${i}`}
+          value={annotations[i]}
+          placeholder=""
+          onChange={handleChange}
+          invalid={invalidFields.indexOf(`annotation${i}`) > -1}
+          autoComplete="off"
+          disabled={disabled}
+        />
+        <div key={`colon${i}`} className="colon">
```

Comment:
Do we need a separate div for a colon? Can we achieve the same thing via styles instead using `::after { content: ":"; }`?

Reply:
ill try to see if it works that way. But the main reason behind the div was for accurate spacing between the other two inputs